src/main/java/frc/robot/Robot.java

```diff
@@ -59,34 +68,58 @@ public void teleopInit(){
 
   @Override
   public void teleopPeriodic() {
-    double leftDrive = -Drive.getTurnPower() * .2;
-    double rightDrive =  Drive.getForwardPower() * .35;   
-    Shift = Drive.driveStick.getRawButton(9); 
+    double leftDrive = -Drive.getTurnPower();
+    double rightDrive =  Drive.getForwardPower();   
+    Shift = Drive.driveStick.getRawButton(5); 
     //This makes the robot drive | Turn power is multiplied by .3 to make it slower and drive is by .5 to make is slower as well
-   Drive.arcadeDrive((-1 * Drive.getTurnPower()) * .2, Drive.getForwardPower() * .35, Shift);
-   CurrentPool.currentPool();
+    Drive.arcadeDrive(Drive.getTurnPower(), Drive.getForwardPower(), Shift);
+    Drive.Climber();
+      // if (Drive.driveStick.getRawButton(1)) {
+      //   Drive.R1.set(.5);
+      // } else if (Drive.driveStick.getRawButton(2)) {
+      //   Drive.R2.set(.5);
+      // } else if (Drive.driveStick.getRawButton(3)) {
+      //   Drive.L1.set(-.5);
+      // } else if (Drive.driveStick.getRawButton(4)) {
+      //   Drive.L2.set(-.5);
+      // } else {
+      //   Drive.R1.set(0);
+      //   Drive.R2.set(0);
+      //   Drive.L1.set(0);
+      //   Drive.L2.set(0);
+      // }
+  //  CurrentPool.currentPool();
```

Comment:
What is happening here with all these comments? Are these meant to be commented out? We should try to remove unused or commented code...

That said it looks like some of it might be crucial to the functionality of the robot? Should it be commented? If it is causing an issue/conflicts with the new code you commit, you'll need to come up with a way to ensure it doesn't break the existing functionality of the robot code...